tests/functional/spec/features/placeholder/playback_confirmation.spec.js

```diff
@@ -10,8 +10,6 @@ describe("Feature: Playback Confirmation", () => {
     await $(MandatoryCheckboxPage.ham()).click();
     await click(MandatoryCheckboxPage.submit());
 
-    await expect(await $("#confirm-answers-question ul").getHTML())
-      .to.contain("Cheese")
-      .to.contain("Ham");
+    await expect(await $("#confirm-answers-question ul").getHTML()).toContain("Ham");
```

Comment:
Looks like the tool didn't handle a chained replacement here, do you remember having to deal with this one manually? if not might be worth searching for any other occurences

Reply:
I don't remember having to do it manually, I think the tool removed it. Yeah, I can have look at that